store/src/java/com/zimbra/cs/account/ProvUtil.java

```diff
@@ -1647,6 +1650,9 @@ private boolean execute(String args[]) throws ServiceException, ArgException, IO
         case GET_HAB_GROUP_MEMBERS:
             doGetHABGroupMembers(args);
             break;
+        case SEND_MDM_EMAIL:
+            console.println(prov.sendMdmEmail(args[1], args[2]));
```

Comment:
I didn't get it? what is use of console here?

Reply:
console was needed to check if the mail was getting sent properly was added for debugging purpose.
will remove it.